### ISSUE TITLE
Bower install error: `breakpoint` file not found

### ISSUE THREAD

**Issue by chrisspiegl** [at-import/Singularity#144]
The install via bower works nicely except if I import singularitygs it fails to find the breakpoint import.

Here is what I did:

```
@import "../bower_components/singularity/stylesheets/singularitygs";
```

The error message is:

```
singularitygs:4: error: file to import not found or unreadable: "breakpoint"
```

The reason for this is that bower does not install dependencies inside the package folder (like npm does) but it installes all the dpendencies in the same folder `bower_components` so the singularity `@import 'breakpoint'` fails.

**Comment by Snugug** [comment]
Yup. You can't use Singularity or Breakpoint through Bower quite yet. They both at the moment require some Ruby which makes them only usable through Compass and their respective Compass gems. Once we update both to Sass 3.3, you will be able to use them as proper Bower projects, but only with a compiler that is compatible with Sass 3.3 (at the time of this writing, libsass which powers grunt-sass is neither fully compatible with Sass 3.2 nor 3.3).

**Comment by Snugug** [comment]
Resolved in #157  (which will become v1.2.0 of Singularity)
